### PR TITLE
docs: Add Standard Detection page

### PR DESCRIPTION
**Pull request by YamenMerhi**
## What does this PR introduce?
- New Page in the **[Standards Section](https://docs.lukso.tech/standards/introduction)** after the introduction page that:
     - Lists the **two types** of standards at LUKSO
     - Explains the difference between **InterfaceId** and **SupportedStandards:XYZ Key**

### CODE REVIEW

File: docs/standards/standard-detection.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
+---
+title: 'Standard Detection'
+sidebar_position: 2
+---
+
+# Standard Detection
+
+:::caution
+
+The **InterfaceId** and the **SupportedStandards:XYZ** key is not the best secure way to check for a standard, as they could be set manually.
```
Comment (Hugoo) [inline docs/standards/standard-detection.md:10]:
```suggestion
The **InterfaceId** and the **SupportedStandards:XYZ** key is not the most secure way to check for a standard, as they could be set manually.
```

File: docs/standards/standard-detection.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+:::
+
+There are 2 types of standards at **LUKSO**:
+
+- **Interface Standards**: Where we standardize a set of functions. i.e: [LSP0-ERC725Account](./universal-profile/01-lsp0-erc725account.md), [LSP6-KeyManager](./universal-profile/04-lsp6-key-manager.md), [LSP7-DigitalAsset](./nft-2.0/02-LSP7-Digital-Asset.md), etc ..
+
+- **Metadata Standards**: Where we standardize a set of ERC725Y keys. i.e: [LSP3-UniversalProfile-Metadata](./universal-profile/03-lsp3-universal-profile-metadata.md), [LSP4-DigitalAsset-Metadata](https://github.com/lukso-network/LIPs/blob/main/LSPs/LSP-4-DigitalAsset-Metadata.md), [LSP10ReceivedVaults](https://github.com/lukso-network/LIPs/blob/main/LSPs/LSP-10-ReceivedVaults.md), etc ..
+
+## Difference between InterfaceId Vs SupportedStandard Key
```
Comment (Hugoo) [inline docs/standards/standard-detection.md:20]:
```suggestion
## Difference between InterfaceId VS SupportedStandard Key
```

or

```suggestion
## Difference between InterfaceId vs SupportedStandard Key
```

File: docs/standards/standard-detection.md [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+
+
+Checking the **InterfaceId** for a contract using [ERC165 standard](https://eips.ethereum.org/EIPS/eip-165) is equal for checking if the contract is implementing a set of functions. As for checking for the **SupportedStandards:XYZ** ERC725Y key, its equal for checking if the contract is implementing **XYZ** related ERC725Y Keys.
```
Comment (Hugoo) [inline docs/standards/standard-detection.md:23]:
```suggestion
Checking the **InterfaceId** for a contract using [ERC165 standard](https://eips.ethereum.org/EIPS/eip-165) is equal for checking if the contract is implementing a set of functions. As for checking for the **SupportedStandards:XYZ** ERC725Y key, it's equal for checking if the contract is implementing **XYZ** related ERC725Y Keys.
```

**Review by Hugoo — APPROVED** [review]

File: docs/standards/standard-detection.md [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+There are 2 types of standards at **LUKSO**:
+
+- **Interface Standards**: Where we standardize a set of functions. i.e: [LSP0-ERC725Account](./universal-profile/01-lsp0-erc725account.md), [LSP6-KeyManager](./universal-profile/04-lsp6-key-manager.md), [LSP7-DigitalAsset](./nft-2.0/02-LSP7-Digital-Asset.md), etc ..
+
+- **Metadata Standards**: Where we standardize a set of ERC725Y keys. i.e: [LSP3-UniversalProfile-Metadata](./universal-profile/03-lsp3-universal-profile-metadata.md), [LSP4-DigitalAsset-Metadata](https://github.com/lukso-network/LIPs/blob/main/LSPs/LSP-4-DigitalAsset-Metadata.md), [LSP10ReceivedVaults](https://github.com/lukso-network/LIPs/blob/main/LSPs/LSP-10-ReceivedVaults.md), etc ..
+
+## Difference between InterfaceId VS SupportedStandard Key
+
+
+Checking the **InterfaceId** for a contract using [ERC165 standard](https://eips.ethereum.org/EIPS/eip-165) is equal for checking if the contract is implementing a set of functions. As for checking for the **SupportedStandards:XYZ** ERC725Y key, it's equal for checking if the contract is implementing **XYZ** related ERC725Y Keys.
```
Comment (CJ42) [inline docs/standards/standard-detection.md:23]:
```suggestion
Checking the **InterfaceId** for a contract using [ERC165 standard](https://eips.ethereum.org/EIPS/eip-165) is equal to checking if the contract implements a set of functions. As for checking for the **SupportedStandards:XYZ** ERC725Y key, it's equal for checking if the contract is implementing **XYZ** related ERC725Y Keys.
```

File: docs/standards/standard-detection.md [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+
+### Example
+
+![Interface and metadata standards](../../static/img/standard-detection.jpeg)
+
+
+**UniversalProfile** contract supports **[ERC725Account InterfaceId](./smart-contracts/interface-ids)**, this means that the contract **SHOULD** implement functions like `getData`, `setData`, `execute`, `universalReceiver`, `isValidSignature` .
```
Comment (CJ42) [inline docs/standards/standard-detection.md:30]:
```suggestion
A **UniversalProfile** contract supports **[ERC725Account InterfaceId](./smart-contracts/interface-ids)**. This means that the contract **SHOULD** implement functions like `getData`, `setData`, `execute`, `universalReceiver`, `isValidSignature` .    
```

**Review by CJ42 — COMMENTED** [review]
added comments

**Pull request merged**